client/webserver/site/src/js/app.js

```diff
@@ -0,0 +1,333 @@
+  bind(sellBttn, 'click', () => swapBttns(buyBttn, sellBttn))
+  const limitBttn = get('limitBttn')
+  const marketBttn = get('marketBttn')
+  const tifDiv = get('tifBox')
+  // const tifCheck = tifDiv.querySelector('input[type=checkbox]')
+  bind(limitBttn, 'click', () => {
+    priceBox.classList.remove('d-hide')
+    tifDiv.classList.remove('d-hide')
+    swapBttns(marketBttn, limitBttn)
+  })
+  bind(marketBttn, 'click', () => {
+    priceBox.classList.add('d-hide')
+    tifDiv.classList.add('d-hide')
+    swapBttns(limitBttn, marketBttn)
+  })
+  const baseImg = get('baseImg')
+  const quoteImg = get('quoteImg')
+  const baseBalance = get('baseBalance')
+  const quoteBalance = get('quoteBalance')
+  const reportPrice = p => { priceField.value = p.toFixed(8) }
+  const reporters = {
+    price: reportPrice
+  }
+  const chart = new DepthChart(get('marketchart'), reporters)
+  const marketList = get('marketList') // includes dex headers
+  const marketRows = marketList.querySelectorAll('.marketrow')
+  const markets = []
+
+  // Check if there is a market saved for the user.
+  var lastMarket = State.fetch('selectedMarket')
+  var mktFound = false
+  marketRows.forEach(div => {
+    const base = parseInt(div.dataset.base)
+    const quote = parseInt(div.dataset.quote)
+    const dex = div.dataset.dex
+    mktFound = mktFound || (lastMarket && lastMarket.dex === dex &&
+      lastMarket.base === base && lastMarket.quote === quote)
+    bind(div, 'click', () => {
+      marketLoader.classList.remove('d-none')
+      setMarket(main, dex, base, quote)
+    })
+    markets.push({
+      base: base,
+      quote: quote,
+      dex: dex
+    })
+  })
+
+  // Get the order row to use as a template.
+  const rowTemplate = get('orderRow')
+  rowTemplate.remove()
+  delete rowTemplate.id
+  const tableBuilder = {
+    row: rowTemplate,
+    buys: get('buyRows'),
+    sells: get('sellRows'),
+    reporters: reporters
+  }
+
+  ws.registerEvtHandler('book', data => {
+    // if (e.market !== market && e.market !== '') return
+    handleBook(main, chart, data, tableBuilder)
+    market = data.market
+    marketLoader.classList.add('d-none')
+    marketRows.forEach(row => {
+      if (row.dataset.dex === data.dex && row.dataset.base === data.base && row.dataset.quote === data.quote) {
+        row.classList.add('selected')
+      } else {
+        row.classList.remove('selected')
+      }
+    })
+    State.store('selectedMarket', {
+      dex: data.dex,
+      base: data.base,
+      quote: data.quote
+    })
+    baseUnits.forEach(el => { el.textContent = data.base })
+    quoteUnits.forEach(el => { el.textContent = data.quote })
+    baseImg.src = `/img/coins/${data.baseSymbol.toLowerCase()}.png`
+    quoteImg.src = `/img/coins/${data.quoteSymbol.toLowerCase()}.png`
+    baseBalance.textContent = formatCoinValue(data.baseBalance/1e8)
+    quoteBalance.textContent = formatCoinValue(data.quoteBalance/1e8)
```

Comment:
The new eslint is demanding spaces around the `/`.  :man_shrugging:

Reply:
I think the old one did this too. I just missed it.